src/hb-set.hh

```diff
@@ -506,16 +506,34 @@ struct hb_set_t
 
   bool is_subset (const hb_set_t *larger_set) const
   {
-    if (get_population () > larger_set->get_population ())
-      return false;
+	if(unlikely(larger_set->page_map.length == 0)) {
+		return false;
+	}
+	if(page_map[page_map.length].major > larger_set->page_map[larger_set->page_map.length].major) {
```

Comment:
Pages may exist that are all zeros, so it isn't valid to return early here.

Reply:
Fixed up the check to ensure small set pages beyond the large set are all empty.